about/policy/sgid/index.md

```diff
@@ -10,16 +10,33 @@ categories: []
 date: 2019-12-04 10:01:27 -0500
 ---
 
-The following applies to data in the internal SGID database.
+The following policies apply to data in the internal SGID Enterprise Geodatabase.
 
-## Adding New Tables
+## Adding New Datasets to the SGID
 {: .text-left }
 
-When adding new data to the SGID, the steps outlined in the [Porter](https://github.com/agrc/porter) project will be followed.
+We strive to achieve and maintain a high standard of quality for the SGID. One of the best ways to do this is making sure all datasets coming into the SGID are high-quality datasets. We use the following policies and procedures to guide this effort.
 
-Additional, related info:
+### Qualifying Datasets
+{: .text-left }
+
+Datasets requested to be added to the SGID by a state or local agency or organization should meet all the following criteria:
```

Comment:
```suggestion
Datasets should meet all the following criteria:
```

Reply:
I was thinking we should leave this narrow and specific to new requests so that the theoretical arguer doesn't claim "well, existing datasets x, y, and z don't meet these, so why won't you accept my new dataset q?" 

Is this too paranoid?

Reply:
I like the shorter sentence. I hope no one will feel the need to argue and if they do we can put them in timeout.